src/App.js

```diff
@@ -38,7 +38,10 @@ firebase.analytics();
 export default function App() {
   function isUserLoggedIn() {
     return false;
-  }
+}
+// toggle for different navbar
+const isAdmin = true; 
```

Comment:
To answer your question, this way of passing isAdmin works great for now. At some point down the road, it will likely be changed down the road into a userObject that contains information like isAdmin.